docs/source/examining-results.rst

```diff
@@ -3,7 +3,7 @@ Examining Results
 =================
 
 
-.. todo:: Overview of figures/summaries generated from results.
```

Comment:
Why is this being removed?

Reply:
The `..todo::` was not allowing the intergration with readthedocs at that time I was desperate to make it work and was removing everything. It was giving some error